libvast/test/compressedbuf.cpp

```diff
@@ -60,7 +63,7 @@ TEST(compressedbuf - iostream interface) {
 #ifdef VAST_HAVE_SNAPPY
   methods.push_back(compression::snappy);
 #endif
-  std::vector<size_t> block_sizes = {1, 2, 64, 256, 1024, 16 << 10};
+  std::vector<size_t> block_sizes = {1, 2, 64, 256, 1_KiB, 16_MiB};
```

Comment:
Was this wrong before or did you mean to write `16_KiB`?

Reply:
Good catch, this should have been `16_KiB`. See, the abstraction already paid off. 😉

(Fixed as of 6882ebf.)